code/Control/UserDefinedFormController.php

```diff
@@ -433,15 +433,21 @@ public function process($data, $form)
                     $submittedFormField = $submittedFields->find('Name', $recipient->SendEmailFromField()->Name);
 
                     if ($submittedFormField && $submittedFormField->Value && is_string($submittedFormField->Value)) {
-                        $email->setReplyTo(explode(',', $submittedFormField->Value ?? ''));
+                        $emailSendTo = $this->validEmailsToArray($submittedFormField->Value);
+                        foreach ($emailSendTo as $emailValue) {
+                            $email->setReplyTo($emailValue);
```

Comment:
Does the email instance actually hold several ReplyTo addresses after this or is it always the last one only?
There must be a reason why `setReplyTo` only takes a single string and `setTo` takes an array. Also seems strange to set multiple emails to `setFrom` field.

Reply:
In previous implementation of `Email` class `setReplyTo` method got either array or string and `Swifmailer` process them after.
https://github.com/silverstripe/silverstripe-framework/blob/4.13/src/Control/Email/Email.php#L566-L578
In new implementation `setReplyTo` method gets only string as a first argument and pass it as parameter to `Adress` class constructor, which also gets only string.
https://github.com/silverstripe/silverstripe-framework/blob/5/src/Control/Email/Email.php#L297-L300

Reply:
If you're iterating over an array of email addresses, instead of using `setReplyTo()` you'll want to use `addReplyTo()`  which defined on `Symfony\Component\Mime\Email` not `SilverStripe\Control\Email\Email` (which itself extends the Symfony Email class)